app/views/user-rights.njk

```diff
@@ -11,7 +11,7 @@
     {% set Manager=true %}
   {% elseif rights === 'System Admin' %}
     {% set SystemAdmin=true %}
-  {% elseif rights === 'Data Admin' %}
+  {% elseif rights === 'Super User' %}
     {% set DataAdmin=true %}
```

Comment:
should be `SuperUser`
actually, should be `superUser`
actually, should be done in the controller!